tests/aequilibrae/matrix/test_aequilibraeMatrix.py

```diff
@@ -43,8 +45,11 @@ def setUp(self) -> None:
 
     def tearDown(self) -> None:
         try:
-            os.remove(name_test) if os.path.exists(name_test) else None
-            self.matrix = None
+            del self.matrix
```

Comment:
Creating a temp directory within tests and then deleting it at the end will save you from having to delete them individually

Reply:
yeah, you can use ```with tempfile.tempDirectory() as temp_dir:``` to do it all in one neat block

Reply:
That is failing in GitHub Actions under Windows....  I tried...  I am deleting the files to not pollute my own system when I am running the tests locally...